src/lib/Behat/Page/ObjectStatePage.php

```diff
@@ -28,24 +29,27 @@ class ObjectStatePage extends Page
     /** @var mixed */
     private $expectedObjectStateId;
 
-    /** @var \Ibexa\AdminUi\Behat\Component\Table\Table */
-    private $table;
-
-    public function __construct(Session $session, Router $router, Repository $repository, TableBuilder $tableBuilder)
+    public function __construct(Session $session, Router $router, Repository $repository)
     {
         parent::__construct($session, $router);
         $this->repository = $repository;
-        $this->table = $tableBuilder->newTable()->build();
     }
 
     public function hasAttribute($label, $value)
     {
-        return $this->table->hasElement([$label => $value]);
+        return $this->getHTMLPage()
+                ->findAll($this->getLocator('objectStateGroupAttribute'))
```

Comment:
```suggestion
                ->findAll($this->getLocator('objectStateAttribute'))
```
This is `ObjectStatePage`, we shouldn't mention `objectStateGroup` here 😉 

Also needs to be adjusted in https://github.com/ibexa/admin-ui/pull/354/files#diff-4bffcba26eaf166c587bf709ff64107b693e7747ad578e4531b621a8c081e1ccR89